src/lib/shared/payloads.ts

```diff
@@ -0,0 +1,57 @@
+import type { TransactionResponseJSON } from '@hashgraph/sdk'
+
+/*
+ * Request params
+ *
+ */
+
+export type SignTransactionAndSendParams = {
+  signerAccountId: string
+  signedTransaction: string
+}
+
+export type SendTransactionOnlyParams = {
+  signedTransaction: string
+}
+
+export type SignTransactionBodyParams = {
+  signerAccountId: string
+  transactionBody: string
+}
+
+export type GetNodeAddressesParams = undefined
+
+export type SignMessageParams = {
+  message: string
+}
+
+export type QueryParams = {
```

Comment:
I think this will also need signerAccountId to define what account should pay for the query